ooniprobe/Utility/OONIApi.m

```diff
@@ -191,6 +213,15 @@ + (void)checkReportId:(NSString*)report_id
     [downloadTask resume];
 }
 
++ (void)checkReportId:(NSString*)report_id
+           onSuccess:(void (^)(BOOL))successcb
+             onError:(void (^)(NSError*))errorcb{
+    [self checkReportId:OONI_API_BASE_URL
```

Comment:
@lorenzoPrimi do we have a test for this function?

Reply:
No, I don't think is needed, was an hotfix to prevent to ddos ouselves again

Reply:
ok